quanta-d51-2u/README.md

```diff
@@ -2,6 +2,7 @@
 
 ## BIOS firmware upgarde
 - The BIOS firmware files should be acquired from the vendor.  The BIOS image and BIOS upgrade executable are then extracted into the /static/bios location of the skupack and the config.json should be updated with the md5sum of the firmware image.
+- A valid BMC or BIOS firmware image should end with .bin, .BIN, .zip, .img or .ima, and only image path with bios/, bmc/ folder or without folder are supported. Other format of "file" value will be recognized as fault by schema validator.

```

Comment:
I have some concern this is not be a good idea to restrict the image path.
Add @zyoung51 for confirm, whether this kind of restriction matches with your design?